chore: update auth docs now that cdk tokens are supported for adminRole and identityPoolId (#6032)
aws-amplify/docs: src/pages/cli/graphql/authorization-rules.mdx

</Block>
<Block name="AWS CDK">

> In your CDK construct, you'll need to enable this "sandbox mode" via an input parameter, rather than as part of the Graphql schema definition:

Reviewer: ```suggestion
In the CDK construct, we call this the "sandbox mode" that you need to explicitly enable via an input parameter. 
```

Designate an IAM role for unauthenticated identities by setting the `iamConfig` property:

```ts
// Note: this sample uses the alpha Cognito Identity Pool construct, but is not required, CfnIdentityPool can be used as well
import cognito_identitypool from '@aws-cdk/aws-cognito-identitypool-alpha';

const identityPool = new cognito_identitypool.IdentityPool(stack, 'MyNewIdentityPool', {
  allowUnauthenticatedIdentities: true,
  authenticationProviders: { userPools: [new cognito_identitypool.UserPoolAuthenticationProvider({

>     userPool: <your_identity_pool>,

Reviewer: this should say "your_user_pool" and same in the next line

Designate an IAM role for authenticated identities by setting the `iamConfig` property:

```ts
// Note: this sample uses the alpha Cognito Identity Pool construct, but is not required, CfnIdentityPool can be used as well
import cognito_identitypool from '@aws-cdk/aws-cognito-identitypool-alpha';

const identityPool = new cognito_identitypool.IdentityPool(stack, 'MyNewIdentityPool', {
  allowUnauthenticatedIdentities: true,
  authenticationProviders: { userPools: [new cognito_identitypool.UserPoolAuthenticationProvider({
    userPool: <your_identity_pool>,
    userPoolClient: <your_identity_pool_client>,
  })] },

> });

Reviewer: same feedback as above
